consensus/spos/worker.go

```diff
@@ -325,6 +325,13 @@ func (wrk *Worker) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedP
 	cnsMsg := &consensus.Message{}
 	err = wrk.marshalizer.Unmarshal(cnsMsg, message.Data())
 	if err != nil {
+		//this situation is so severe that we have to black list both the message originator and the connected peer
+		//that disseminated this message.
+
+		reason := "blacklisted due to invalid consensus message"
```

Comment:
maybe define a constant somewhere

Reply:
used only here, once, will do if I do need it elsewhere